src/scripts/main.js

```diff
@@ -1,7 +1,24 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const bodyElem = document.querySelector('body');
+  const divElem = document.createElement('div');
+  const h2Elem = document.createElement('h2');
+  const pElem = document.createElement('p');
+
+  divElem.style.top = `${posTop}px`;
+  divElem.style.right = `${posRight}px`;
+  divElem.className = `notification ${type}`;
+  h2Elem.className = 'title';
+  h2Elem.innerText = title;
+  pElem.innerText = description;
+  bodyElem.append(divElem);
+  divElem.append(h2Elem);
+  divElem.append(pElem);
```

Comment:
You can just create div and then set inner HTML for it.
It will be more visually clear